go/test/endtoend/backup/vtbackup/backup_only_test.go

```diff
@@ -62,12 +63,21 @@ func TestTabletInitialBackup(t *testing.T) {
 	// Initialize the tablets
 	initTablets(t, false, false)
 
-	// Restore the Tablets
+	vtTabletVersion, err := cluster.GetMajorVersion("vttablet")
+	require.NoError(t, err)
+	// For all version above v15, each replica will start in super-read-only mode.
+	if vtTabletVersion > 16 {
+		err := primary.VttabletProcess.CreateDB("testDB")
+		require.ErrorContains(t, err, "The MySQL server is running with the --super-read-only option so it cannot execute this statement")
+		err = replica1.VttabletProcess.CreateDB("testDB")
+		require.ErrorContains(t, err, "The MySQL server is running with the --super-read-only option so it cannot execute this statement")
```

Comment:
Yes I am piggybacking on existing test. Instead of writing a separate test. I just add this in existing test to check super_read_only functionality. Basically it testing from v17 onwards all replica should come up as super_read_only.